aten/src/ATen/native/ReduceOpsUtils.h

```diff
@@ -73,18 +73,19 @@ inline bool _dimreduce_return_trivial(Tensor &result, const Tensor &self,
   return false;
 }
 
-inline bool _dimreduce_return_trivial_no_ident(Tensor &result, const Tensor &self,
-                                               int64_t dim, bool keepdim, const char *fn_name) {
+inline bool _dimreduce_return_trivial_no_ident(Tensor &result, const Tensor &self, 
+                                               const char *fn_name) {
```

Comment:
kode sebelumnya ada parameter `int64_t dim` dan `bool keepdim` yang tidak dipakai dalam fungsi. Sebaiknya dibiarkan seperti itu, tidak dihapus. Kalau mau cleaning unused parameter dalam code, mending buat issue atau PR baru.

Reply:
Ok, i'll change it, thankyou for advise 🙏